CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 * **Overall**
   * [CHANGE] Revised Kubernetes and OpenShift version-checking logic (inc. changing ERROR message to a WARNING)
+  * [UPGRADE] Update driver persistent volume to `disk.csi.azure.com` in Azure deployment sample as that is the supported driver by AKS for K8s v1.26+
```

Comment:
I'm not sure this qualifies as an "upgrade", seems more like a "change" to me.  And I'm not sure the message is particularly clear.  Maybe something like "[CHANGE] Updated Azure deployment sample to replace storage provisioner deprecated in AKS for K8s v1.26+"